src/mame/machine/st0016.cpp

```diff
@@ -274,6 +275,32 @@ void st0016_cpu_device::vregs_w(offs_t offset, uint8_t data)
 
 	   I/O ports:
 
+		$60-$6f video timing related
+
+		$60 \
+		$61 - H sync start?
+
+		$62 \
+		$63 - H screen start?
+
+		$64 \
+		$65 - H screen end >> 1?
+
+		$66 \
+		$67 - H total
+
+		$68 \
+		$69 - V sync start?
+
+		$6a \
+		$6b - V screen start?
+
+		$6c \
+		$6d - V screen end?
+
+		$6e \
+		$6f - V total
+
```

Comment:
What's the proof of this assumption? 
i.e. how the individual games configs this one up?

Reply:
Most of games are writes 0x1c6 in $66-$67 and 0x105 in $6e-$6f (0x112 at macs, for higher vertical resolution).
$64-$65 and $6c-$6d is display resolution related, higher value for higher resolution.
(ex: mayjisn2 (256x224) writes 0xae in $64-$65 and 0xf8 in $6c-$6d, renju writes 0xc4 and 0x103.)
other registers are seems like blanking period parameters.